src/components/navigation/submenu/styles.js

```diff
@@ -113,27 +113,13 @@ export const PanelLink = styled(Link)`
     padding: 5px 0;
     font-size: 1.25rem;
   `};
-
   span {
-    position: relative;
-    &:after {
-      content: '';
-      height: 1px;
-      width: 100%;
-      position: absolute;
-      bottom: 0;
-      left: 0;
-      background-color: transparent;
-      transition: background-color 0.15s linear;
-    }
+    transition: box-shadow 0.15s ease-in-out;
```

Comment:
This is a different `transition` than previously.

Reply:
Yes, I thought it behaved most like that previously. but I can revert to     
`transition: box-shadow 0.15s linear;`

Reply:
Yup. Keep in mind that it's not a matter of preference, this is the requested design and should not be altered here.

Reply:
👍 Okay, I think I've done this.